### PR TITLE
feat(inbox): Callback for when Inbox was synced via forceContentUpdat…

### PR DESCRIPTION
**Pull request by Enuviel**
feat(inbox): Callback for when Inbox was synced via forceContentUpdate(LP-6295).

### CODE REVIEW

**Comment by Enuviel** [comment]
test this please

**Comment by milos1290** [comment]
test this please

**Comment by Enuviel** [comment]
test this please

**Comment by Enuviel** [comment]
test this please

File: AndroidSDK/src/com/leanplum/LeanplumInbox.java [hunk 1]
```diff
@@ -241,6 +258,15 @@ void triggerChanged() {
     }
   }
 
+  void triggerInboxSyncedWithStatus(boolean success) {
```
Comment (milos1290) [inline AndroidSDK/src/com/leanplum/LeanplumInbox.java]: Javadocs

File: AndroidSDK/src/com/leanplum/LeanplumInbox.java [hunk 2]
```diff
@@ -178,6 +181,20 @@ public void removeChangedHandler(InboxChangedCallback handler) {
     }
   }
 
+  // Add a callback for when the forceContentUpdate was called.
+  public void addSyncedHandler(InboxSyncedCallback handler) {
```
Comment (milos1290) [inline AndroidSDK/src/com/leanplum/LeanplumInbox.java]: Add proper javadocs for public API

**Comment by Enuviel** [comment]
test this please

File: AndroidSDK/src/com/leanplum/LeanplumInbox.java [hunk 3]
```diff
@@ -178,6 +181,20 @@ public void removeChangedHandler(InboxChangedCallback handler) {
...
+    synchronized (syncedCallbacks) {
+      syncedCallbacks.add(handler);
+    }
+  }
+
+  // Removes a inbox synced callback.
+  public void removeSyncedHandler(InboxSyncedCallback handler) {
```
Comment (milos1290) [inline AndroidSDK/src/com/leanplum/LeanplumInbox.java]: Add proper javadocs for public API

**Comment by milos1290** [comment]
test this please

**Comment by Enuviel** [comment]
test this please

**Comment by Enuviel** [comment]
test this please

**Comment by Enuviel** [comment]
test this please

**Comment by Enuviel** [comment]
test this please

**Comment by Enuviel** [comment]
test this please

**Pull request merged**